docs/data-sources/shares.md

```diff
@@ -0,0 +1,38 @@
+---
```

Comment:
Should this be in a different PR?

Reply:
this is just the result of "make docs". so it means current main is not up to date.